tests/cypress/integration/pdv_hoc_site_spec.js

```diff
@@ -0,0 +1,487 @@
+describe('Test H.O.C.', () => {
+
+  it('Coveo search', () => {
+    // out of scope
+    // not testable in local environments
+  })
+
+  it('Google Analytics', () => {
+    // out of scope
+    // not testable in local environments
+  })
+
+  it('Qualtrics feedback', () => {
+    // out of scope
+    // not testable in local environments
+  })
+
+  it('Classic/V1 release notes (en-US)', () => {
+    cy.visit('en-us/Content/Topics/ReleaseNotes/okta-relnotes.htm')
+
```

Comment:
let's add something to check that we loaded the release notes. Maybe  a couple of things
- Verify that the title in the page is "Release Notes"
- Verify that we have 9 visible tiles/cards

Reply:
Ah yes, here and elsewhere, I had intended to ask about this. 
I think rather than checking for text, which is changeable, we can check for the presence of structures. I'd like to apply `data-cypress="somename"` attributes to interesting elements, in the source files, and then check for the presence of those elems in output.
Incidentally, some of our interesting elems are not easy to get at, leading to convoluted test instructions. I think these elems could be made easier to find -- and thus, tests easier to read -- by applying `data-cypress` attrs to them.
???